### PR TITLE
Home page: remove country list #8061

### LINKED ISSUE #8061: Home page: remove country list
Current:
image

Suggested:
>TEAMMATES community is growing fast, spanning over 1000 universities from across the globe.
>
>[Who is using TEAMMATES?]()

Reason: Our country list has grown a lot over the recent times and it is not viable to give a short representative list here.

### PR DESCRIPTION
**Pull request by fdigne**
Fixes #8061

### CODE REVIEW

File: .tern-project [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (whipermr5) [inline .tern-project]: Please revert this file as it's an unrelated change.

Comment (fdigne) [reply .tern-project]: Hello, 

How can I do to revert this file ?
